seerep-srv/seerep-server/src/fb-cameraintrinsics-service.cpp

```diff
@@ -0,0 +1,40 @@
+#include "seerep-server/fb-cameraintrinsics-service.h"
+
+namespace seerep_server
+{
+FbCameraIntrinsicsService::FbCameraIntrinsicsService(std::shared_ptr<seerep_core::Core> seerepCore)
+  : ciCore(std::make_shared<seerep_core::CoreCameraIntrinsics>(seerepCore))
+{
+}
+grpc::Status FbCameraIntrinsicsService::GetCameraIntrinsics(
+    grpc::ServerContext* context, const flatbuffers::grpc::Message<seerep::fb::cameraIntrinsicsQuery>* request,
+    grpc::ServerWriter<flatbuffers::grpc::Message<seerep::fb::CameraIntrinsics>>* writer)
+{
+  (void)context;  // ignore this variable without causing warnings
+  auto requestRoot = request->GetRoot();
+
+  // TODO check both project and camera intrinsics uuid are set
+
+  seerep_core_msgs::camera_intrinsics_query ciq_coremsg;
```

Comment:
please use try-catch in both services to catch potential errors. Check e.g. fb-image-service.cpp